samples/nrf9160/modem_shell/src/link/link.c

```diff
@@ -197,6 +212,28 @@ static void link_registered_work(struct k_work *unused)
 
 /******************************************************************************/
 
+static void link_cell_change_work(struct k_work *work_item)
+{
+	struct cell_change_data *data = CONTAINER_OF(work_item, struct cell_change_data, work);
+	struct lte_xmonitor_resp_t xmonitor_resp;
+	int ret = link_api_xmonitor_read(&xmonitor_resp);
+
+	if (ret) {
+		mosh_print(
+			"LTE cell changed: ID: %d, Tracking area: %d",
+			data->cell_id, data->tac);
+		return;
+	}
+	mosh_print(
+		"LTE cell changed: ID: %d (0x%s), PCI %d, Tracking area: %d (0x%s), Band: %d, RSRP: %d: %ddBm, SNR %d: %ddB",
```

Comment:
RSRP and SNR are printed a little bit differently. How about something like this? The raw value and dBm/dB can be the other way around as well.
```suggestion
		"LTE cell changed: ID: %d (0x%s), PCI %d, Tracking area: %d (0x%s), Band: %d, RSRP: %d (%ddBm), SNR: %d (%ddB)",
```

Reply:
changed